AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -564,15 +564,14 @@ - (void)relayoutAllRows
     LOG(@"Edit Command - relayoutRows");
     [_editingTransactionQueue waitUntilAllOperationsAreFinished];
     
-    NSArray *indexPaths = ASIndexPathsForMultidimensionalArray(_completedNodes);
-    NSArray *loadedNodes = ASFindElementsInMultidimensionalArrayAtIndexPaths(_completedNodes, indexPaths);
-    
-    for (NSUInteger i = 0; i < loadedNodes.count && i < indexPaths.count; i++) {
-      ASSizeRange constrainedSize = [_dataSource dataController:self constrainedSizeForNodeAtIndexPath:indexPaths[i]];
-      ASCellNode *node = loadedNodes[i];
-      [node measureWithSizeRange:constrainedSize];
-      node.frame = CGRectMake(0.0f, 0.0f, node.calculatedSize.width, node.calculatedSize.height);
-    }
+    [_completedNodes enumerateObjectsUsingBlock:^(NSMutableArray *section, NSUInteger sectionIndex, BOOL *stop) {
```

Comment:
This looks a lot nicer.  I'd actually be really curious to profile both versions on a device.  Thinking about this, I'm not 100% confident which will be faster.  My bet is on the new version, but because of the individual creation of NSIndexPaths, it might not be by much (main win is avoiding the array allocations).

Someday in the magical distant future, it would be sweet to banish NSIndexPath and use ASIndexPath everywhere :).  Not important now, but it may well be in a few months with the new use cases starting to expose this code to large data sets.